identity/src/main/java/com/stripe/android/identity/ui/CameraScreenLaunchedEffectLight.kt

```diff
@@ -0,0 +1,106 @@
+package com.stripe.android.identity.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.NavController
+import com.stripe.android.identity.analytics.IdentityAnalyticsRequestFactory
+import com.stripe.android.identity.ml.FaceDetectorOutput
+import com.stripe.android.identity.ml.IDDetectorOutput
+import com.stripe.android.identity.navigation.CouldNotCaptureDestination
+import com.stripe.android.identity.navigation.navigateTo
+import com.stripe.android.identity.networking.models.VerificationPage
+import com.stripe.android.identity.states.IdentityScanState
+import com.stripe.android.identity.states.IdentityScanState.Companion.isBack
+import com.stripe.android.identity.states.IdentityScanState.Companion.isFront
+import com.stripe.android.identity.viewmodel.IdentityScanViewModel
+import com.stripe.android.identity.viewmodel.IdentityViewModel
+import kotlinx.coroutines.launch
+
+/**
+ * [CameraScreenLaunchedEffect] without checking pageAndModelFiles from IdentityViewModel.
+ */
+@Composable
+internal fun CameraScreenLaunchedEffectLight(
+    identityViewModel: IdentityViewModel,
+    identityScanViewModel: IdentityScanViewModel,
+    verificationPage: VerificationPage,
+    navController: NavController
+) {
+    val lifecycleOwner = LocalLifecycleOwner.current
+    LaunchedEffect(identityScanViewModel) {
```

Comment:
can the identityScanViewModel instance change? just making sure we're not retriggering observers more than needed.